sendinblue.php

```diff
@@ -187,11 +186,16 @@ public function callhookRegister()
             $this->months = Tools::getValue('months');
             $this->years = Tools::getValue('years');
             $birthday = ($this->years . '-' . $this->months . '-' . $this->days);
-            
+
+            echo '<script type="text/javascript">
```

Comment:
Javascript code need to be placed in a js file and located in yourmodule/views/js/yourjsfile.js also you should use the addJs function to add it in your module.  
`$this->context->controller->addJs($this->_path . 'js/newsletter.js');`